{
  "comments": [
    {
      "key": {
        "uuid": "1a756307_ffe92cfd",
        "filename": "private/domain.te",
        "patchSetId": 9
      },
      "lineNbr": 6,
      "author": {
        "id": 1059554
      },
      "writtenOn": "2016-12-15T22:40:37Z",
      "side": 1,
      "message": "crash_dump should not be allowed to ptrace itself",
      "revId": "ffa2b88ff604317e6585665170c3a6314e4dba8a",
      "serverId": "85c56323-6fa9-3386-8a01-6480fb634889",
      "unresolved": false
    },
    {
      "key": {
        "uuid": "1a756307_bfa89473",
        "filename": "public/crash_dump.te",
        "patchSetId": 9
      },
      "lineNbr": 8,
      "author": {
        "id": 1059554
      },
      "writtenOn": "2016-12-15T22:40:37Z",
      "side": 1,
      "message": "crash_dump should not be able to ptrace itself",
      "revId": "ffa2b88ff604317e6585665170c3a6314e4dba8a",
      "serverId": "85c56323-6fa9-3386-8a01-6480fb634889",
      "unresolved": false
    }
  ]
}